examples/stable-diffusion/training/README.md

```diff
@@ -163,86 +163,92 @@ Install the requirements:
 pip install -r requirements.txt
 ```
 
-### Example for SDXL
-We can launch the fine-tuning of SDXL model using:
+### Single-card Training
 
 ```bash
 python train_text_to_image_sdxl.py \
   --pretrained_model_name_or_path stabilityai/stable-diffusion-xl-base-1.0 \
   --pretrained_vae_model_name_or_path stabilityai/sdxl-vae \
   --dataset_name lambdalabs/pokemon-blip-captions \
-  --resolution 1024 \
+  --resolution 512 \
+  --crop_resolution 512 \
   --center_crop \
   --random_flip \
   --proportion_empty_prompts=0.2 \
-  --train_batch_size 1 \
-  --gradient_accumulation_steps 4 \
-  --max_train_steps 3000 \
+  --train_batch_size 16 \
+  --max_train_steps 2500 \
   --learning_rate 1e-05 \
   --max_grad_norm 1 \
   --lr_scheduler constant \
   --lr_warmup_steps 0 \
   --output_dir sdxl-pokemon-model \
   --gaudi_config_name Habana/stable-diffusion \
   --throughput_warmup_steps 3 \
-  --use_hpu_graphs \
-  --bf16
+  --dataloader_num_workers 8 \
+  --bf16 \
+  --use_hpu_graphs_for_training \
+  --use_hpu_graphs_for_inference \
+  --validation_prompt="a robotic cat with wings" \
+  --validation_epochs 48 \
+  --checkpointing_steps 2500 \
+  --logging_step 10 \
+  --adjust_throughput
 ```
 
-### Example for LoRA SDXL
-
-Low-Rank Adaption (LoRA) allows adapting a pretrained model by adding pairs of rank-decomposition matrices to
-existing weights and only training those newly added weights.
 
-We can launch the LoRA based fine-tuning of SDXL model using:
+### Multi-card Training
+```bash
+PT_HPU_RECIPE_CACHE_CONFIG=/tmp/stdxl_recipe_cache,True,1024  \
+python ../../gaudi_spawn.py --world_size 8 --use_mpi train_text_to_image_sdxl.py \
+  --pretrained_model_name_or_path stabilityai/stable-diffusion-xl-base-1.0 \
+  --pretrained_vae_model_name_or_path stabilityai/sdxl-vae \
+  --dataset_name lambdalabs/pokemon-blip-captions \
+  --resolution 512 \
+  --crop_resolution 512 \
+  --center_crop \
+  --random_flip \
+  --proportion_empty_prompts=0.2 \
+  --train_batch_size 16 \
+  --max_train_steps 336 \
```

Comment:
Does 336 steps correspond to one or several epochs?

Reply:
several epochs

Reply:
Okay, but a specific number of epochs? If yes, it would be better to use `--num_train_epochs`